english/DC-Comics/Man/melancholic.md

```diff
@@ -0,0 +1,14 @@
+The scene opens with an unshaven man in his bathrobe.  He lounges on his couch next to a mostly-empty pizza box.  The phone rings; he answers but we only hear his side of the conversation...
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.